initdb.py

```diff
@@ -67,107 +65,125 @@
 SQL_INSERT['locations'] = "INSERT INTO locations " \
                           "VALUES(%s, %s, %s, %s);"
 
-# Load the data sets
-DATASETS = dict()
-DATASETS['income'] = csv.reader(open(os.path.join('datasets', 'income-data.csv')))
-DATASETS['starbucks'] = csv.reader(open(os.path.join('datasets', 'starbucks.csv')))
-DATASETS['diversity'] = csv.reader(open(os.path.join('datasets', 'diversity.csv')))
-DATASETS['locations'] = csv.reader(open(os.path.join('datasets', 'locations.csv')))
-
+# Load each data set
+DATASETS = {}
+DATASETS['income'] = csv.reader(open(os.path.join('datasets',
+                                                  'income-data.csv')))
+DATASETS['starbucks'] = csv.reader(open(os.path.join('datasets',
+                                                     'starbucks.csv')))
+DATASETS['diversity'] = csv.reader(open(os.path.join('datasets',
+                                                     'diversity.csv')))
+DATASETS['locations'] = csv.reader(open(os.path.join('datasets',
+                                                     'locations.csv')))
+
+
+def init_connections(username=None, password=None):
```

Comment:
Oh shoot I didn't realize it pushed my dbmanager changes too. I wasn't near finished with dbmanager